docs/cli/cli-orders.md

```diff
@@ -515,7 +515,7 @@ One cool little trick is that you can even stream in the JSON directly with `cur
 
 ```console
 curl -s https://raw.githubusercontent.com/planetlabs/planet-client-python/main/docs/cli/request-json/tools-clip-composite.json \
-| planet orders request --item-type PSScene --bundle analytic_sr_udm2 --name 'Streaming Clip & Composite' \
+| planet orders request --item-type PSScene --bundle analytic_sr_udm2 --name 'Streaming Clip & Composite' --no-stac \
```

Comment:
Note that I had to add a `--no-stac` flag here because STAC output wit the composite tool is not currently supported.

Reply:
Though I do realize that this is before we start talking about STAC output, which is the next section. Thoughts on keeping this example here? @JuLeeAtPlanet @cholmes?